### DISCUSSION TITLE
assertion failed: terminate called without an active exception

### DISCUSSION THREAD

**Discussion by kanli201904** [gabime/spdlog#2570] [Q&A]
Hello: 
I met one error :
spdlog/fmt/bundled/format.h:2070:assertion failed: terminate called without an active exception
Is there some suggestion for solving the problem?
thanks

**Comment by tt4g** [comment]
`spdlog/fmt/bundled/format.h` is copied from the fmt library, so you should ask in [fmt repository](https://github.com/fmtlib/fmt).

And it does not seem to be the source code of the latest version (spdlog v1.11.0). So we cannot even guess the cause from the error message.

**Reply by tt4g** [reply]
I found an assertion code (cause of error) in spdlog v1.10.0 branch (fmt version 8.1.1).

https://github.com/gabime/spdlog/blob/76fb40d95455f249bd70824ecfcae7a8f0930fa3/include/spdlog/fmt/bundled/format.h#L2067-L2072

It seems to be an assertion code that always fails.
Either there is a bug in the fmt library or there is a problem in your project code.

**Reply by kanli201904** [reply]
Thanks, 
that's  spdlog V1.8.1 ,  i will try the lastest version
